pkgs/tools/admin/proxmove/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "proxmove";
+  version = "1.2";
```

Comment:
Set an explicit format.

Reply:
Am I misremembering that "setuptools" is already default for `format` here? Convention is to explicit it regardless while the later-mentioned `doCheck` should rely on the implicitness of its default `true` value?

Reply:
Ah excellent! Thanks :)